index.html

```diff
@@ -0,0 +1,559 @@
+                    <h5 class="deal">
+                        <span class="old-price">$1495</span><span class="new-price">$59</span>
+                    </h5>
+                </div>
+                <div class=latest-deals-6>
+                    <img class="background-discount" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/deals-ribbon.svg" alt="background-discount">
+                    <div class="discount">96% off</div>
+                    <h4 class="latest-deals-h4"><a href="#">The Ultimate Lean Certification Prep Bundle</a></h4>
+                    <h5 class="deal">
+                        <span class="old-price">$1697</span><span class="new-price">$69.99</span>
+                    </h5>
+                </div>
+                <div class=latest-deals-7>
+                    <img class="background-discount" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/deals-ribbon.svg" alt="background-discount">
+                    <div class="discount">98% off</div>
+                    <h4 class="latest-deals-h4"><a href="#">The 2020 Learn to Code Full Stack Developer Certification Bundle</a></h4>
+                    <h5 class="deal">
+                        <span class="old-price">$1650</span><span class="new-price">$38.99</span>
+                    </h5>
+                </div>
+                <div class=latest-deals-8>
+                    <img class="background-discount" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/deals-ribbon.svg" alt="background-discount">
+                    <div class="discount">97% off</div>
+                    <h4 class="latest-deals-h4"><a href="#">FastestVPN: Lifetime Subscription (5 Devices)</a></h4>
+                    <h5 class="deal">
+                        <span class="old-price">$600</span><span class="new-price">$19.99</span>
+                    </h5>
+                </div>
+            </article>
+        </section>
+    </main>
+    <footer>
+        <div class="footer-links">
+            <ul class="social-links">
+                <li><a href="#"><i class="fab fa-facebook-square"></i></a></li>
+                <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fab fa-youtube"></i></a></li>
+                <li><a href="#"><i class="fab fa-instagram"></i></a></li>
+                <li><a href="#"><i class="fab fa-pinterest-p"></i></a></li>
+                <li><a href="#"><i class="fas fa-rss-square"></i></a></li>
+                <li><a href="#"><i class="far fa-envelope"></i></a></li>
+            </ul>
+            <ul class="menu-links">
+                <li><a href="#">Events</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Advertise</a></li>
+                <li><a href="#">Jobs</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
```

Comment:
- [ ] The elements in this list are not displaying in the footer. Check why.
![Screenshot from 2020-03-16 12-02-48](https://user-images.githubusercontent.com/51926937/76787247-37ec6180-677e-11ea-9bca-fe60568c284c.png)